changelog.d/917.change.rst

```diff
@@ -0,0 +1 @@
+``setup.py install`` and ``easy_install`` commands are now officially deprecated. Use other standards-based installers (like pip) and builders (like build). Workloads reliant on this behavior should pin to this major version of Setuptools.
```

Comment:
I wonder if it'd be useful to link https://blog.ganssle.io/articles/2021/10/setup-py-deprecated.html from the changelog.

Reply:
Done in 7b5158b10.